contracts/TokenHub.sol

```diff
@@ -522,6 +522,32 @@ contract TokenHub is ITokenHub, System, IParamSubscriber, IApplication, ISystemR
     return elements.encodeList();
   }
 
+  /**
+   * @dev request a BC token aridrop transfer from BSC
+   *
+   * @param tokenSymbol The token symbol on BSC.
+   * @param recipient The destination address of the transfer on BSC.
+   * @param amount The amount to transfer
+   */
+  function unlock(bytes32 tokenSymbol, address recipient, uint256 amount) external override onlyInit onlyAirDrop payable {
+    uint256 convertedAmount;
+    if (tokenSymbol != BEP2_TOKEN_SYMBOL_FOR_BNB) {
+      address contractAddr = bep2SymbolToContractAddr[tokenSymbol];
+      require(contractAddr != address(0x00), "invalid symbol");
+      uint256 bep20TokenDecimals=bep20ContractDecimals[contractAddr];
+      convertedAmount = convertFromBep2Amount(amount, bep20TokenDecimals);// convert to bep2 amount
+      require(bep20TokenDecimals>=BEP2_TOKEN_DECIMALS || (bep20TokenDecimals<BEP2_TOKEN_DECIMALS && convertedAmount>amount), "amount is too large, uint256 overflow");
```

Comment:
Can you take a look at function `_checkAndLockTransferIn`, I would like to lock the funds for a while if it is  a big transfer.

Reply:
added